src/vue/modules/create-asset-sale/index.vue

```diff
@@ -0,0 +1,53 @@
+<template>
```

Comment:
Why did you create this wrapper? Create single `create-asset-sale-form` module without creating this wrapper

Reply:
ok

Reply:
ok